src/collection.ts

```diff
@@ -101,6 +100,14 @@ export interface ModifyResult<TSchema = Document> {
   ok: 0 | 1;
 }
 
+/** @public */
+export interface CountDocumentsOptions extends AggregateOptions {
+  /** The number of documents to skip. */
+  skip?: number;
+  /** The maximum amounts to count before aborting. */
```

Comment:
```suggestion
  /** The maximum amount of documents to consider. */
```

Reply:
I wonder if aborting referred to some old cursor lingo. Updated!